x/evm/types/key.go

```diff
@@ -0,0 +1,10 @@
+package types
+
+const (
+	// module name
+	ModuleName = "ethermint"
+
+	// TODO: Use this
+	// StoreKey to be used when creating the KVStore
+	StoreKey = ModuleName
```

Comment:
Just to clear up between us, the StoreKey would just be used for our Keeper impl, the EthermintApp when that KV store is set up and the missing AppModule function implementations not included in this PR, right? Or is there somewhere else that should use this

Reply:
Yeah thats the primary usage. It seems to be used to ensure we are always accessing the correct store to just pass the name of the store around (for modularity's sake?). Notably it's passed into the `AppModuleBasic` funcs discussed above.